src/packages/@ncigdc/utils/ajax/index.js

```diff
@@ -1,26 +1,31 @@
 // @flow
-import { CALL_API } from 'redux-api-middleware';
+import { RSAA } from 'redux-api-middleware';
```

Comment:
Updating this dependency to the currently latest version for great good 💰

Reply:
If you like this, you should checkout `createAsyncThunk` from redux-toolkit.  it's beautiful.

Reply:
Personally, not a fan of `redux-api-middleware`; but I hadn't seen `createAsyncThunk` and it looks like a thing of beauty!